.github/workflows/build_portable_linux_pytorch_wheels.yml

```diff
@@ -35,8 +35,14 @@ on:
   workflow_dispatch:
     inputs:
       amdgpu_family:
-        required: true
-        type: string
+        type: choice
+        options:
+          - gfx110X-dgpu
+          - gfx1151
+          - gfx120X-all
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
```

Comment:
We could also

1. keep this as a freeform `string` field
2. add xfail families from https://github.com/ROCm/TheRock/blob/main/build_tools/github_actions/amdgpu_family_matrix.py

I think limiting to only families that we have here makes sense though: https://github.com/ROCm/TheRock/blob/231dd86eed1d1188bdeb4b3f2bdeebf4b59cdbb8/build_tools/third_party/s3_management/manage.py#L38-L44

(but we will want to expand that list, and now we have multiple locations to update)